src/components/WikiResults.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useEffect } from 'react';
+
+import Spinner from './Spinner';
+import ErrorMessage from './ErrorMessage';
+
+import Map from './Map';
+
+const WikiResults = () => {
+  const [data, setData] = useState([]);
+  const [status, setStatus] = useState('idle');
+  const [error, setError] = useState(null);
+
+  const isLoading = status === 'loading';
+  const isSuccess = status === 'success';
+  const isError = status === 'error';
+
+  const { lat, lng } = { lat: 48.8566, lng: 2.3522 };
+
+  useEffect(() => {
```

Comment:
**_Only suggestion (There is not pressure):_**

-  What do you think about creating a custom hook in this case?

Reply:
I like @alxmcr 's suggestion...It would be good to have this be at a higher level so you can pass the data down to both the Map and the List components.